src/ansys/geometry/core/tools/prepare_tools.py

```diff
@@ -295,3 +297,93 @@ def enhanced_share_topology(
             share_topo_response.repaired,
         )
         return message
+
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def find_logos(
+        self, bodies: list["Body"] = None, min_height: Real = None, max_height: Real = None
+    ) -> "LogoProblemArea":
+        """Detect logos in geometry.
+
+        Detects logos, using a list of bodies if provided.
+        The logos are returned as a list of faces.
+
+        Parameters
+        ----------
+        bodies : list[Body], optional
+            List of bodies where logos should be detected
+        min_height : real, optional
+            The minimum height when searching for logos
+        max_height: real, optional
+            The minimum height when searching for logos
+
+        Returns
+        -------
+        Logo problem area
+
+            Problem area with logo faces.
+        """
+        from ansys.geometry.core.designer.body import Body
+
+        # Verify inputs
+        if bodies:
+            if len(bodies) > 0:
```

Comment:
```suggestion
        if bodies and len(bodies) > 0:
```
Multiple nested ifs into a single statement

Reply:
Addressed